src/sinks/aws_cloudwatch_logs/mod.rs

```diff
@@ -160,12 +160,11 @@ impl CloudwatchLogsSinkConfig {
 #[typetag::serde(name = "aws_cloudwatch_logs")]
 impl SinkConfig for CloudwatchLogsSinkConfig {
     fn build(&self, cx: SinkContext) -> crate::Result<(super::RouterSink, super::Healthcheck)> {
-        let batch = self.batch.use_size_as_events()?.get_settings_or_default(
-            BatchSettings::default()
-                .bytes(1_048_576)
-                .events(10_000)
-                .timeout(1),
-        );
+        let batch = BatchSettings::default()
+            .bytes(1_048_576)
```

Comment:
If we wanted to get fancy, we could add `megabytes`/`kilobytes` methods :smile:

Reply:
But would that be `megabytes` or `mebibytes`? :grimacing: